### ISSUE TITLE
Can't update to version 1.0.3

### ISSUE THREAD

**Issue by macguydvd** [tig-nl/persistentshoppingcart-magento2#3]
**To Reproduce**
1. Go to Magento 2 root folder
2. Run command 'composer require tig/persistent-shopping-cart-magento2:1.0.3'

**Expected result**
Latest update of module is installed.

**Actual result**
Composer returns an error:
'tig/persistent-shopping-cart-magento2 1.0.3 requires tig/core 1.0.* -> no matching package found.'

**Workaround**
None that I know of.

**Please complete the following information**
- Magento version: 2.3.3
- PHP version: 7.2

**Comment by tig-dennisvanderhammen** [comment]
Hi @macguydvd,

I'll pass this on to the right person in our team. As a work-around you could try installing version 1.0.2.

**Comment by macguydvd** [comment]
Hello @tig-dennisvanderhammen I do have 1.0.2 installed, but it doesn't show up in the backend at all so I have no way to turn it on as described in the instructions. I believe the cookie is being made, but have no _Persistent Shopping Cart_ option under _Stores > Configuration > Customers_. I was hoping this was the fix.

**Comment by tig-svenluiks** [comment]
Hi @macguydvd ,
If you run composer require tig/persistent-shopping-cart-magento2 in terminal it will install version ^1.0 and it will show up under Stores > Configuration > Customers

**Comment by macguydvd** [comment]
Hello @tig-svenluiks ,
I did install this with Composer and this does not show up under Stores > Configuration > Customers. The installed version was 1.0.2, but composer tells me that 1.0.3 is available. When I run composer upgrade it doesn't update to 1.0.3.
Thanks for any help!

**Comment by tig-svenluiks** [comment]
Good morning @macguydvd ,
It's better to install the module this way:

1. php bin/magento maintenance:enable
2. composer require tig/persistent-shopping-cart-magento2 
3. php bin/magento module:enable tig/persistent-shopping-cart-magento
4. php bin/magento cache:flush
5. php bin/magento setup:upgrade
6. php bin/magento indexer:reindex

Hope it helps.

**Comment by tig-svenluiks** [comment]
Hey @macguydvd ,
Could you try to run composer require tig/persistent-shopping-cart-magento2 1.0.2 That installed it for me and it also showed up in Magento Stores > Configuration > Customer. Could you let me know if it works?

**Comment by tig-svenluiks** [comment]
Hey @macguydvd,
Maybe flushing the cache will make the module appear in Magento.

**Comment by JKetelaar** [comment]
I'm facing the same issue. No Persistent Shopping Cart setting for me either.
![image](https://user-images.githubusercontent.com/3681904/72469386-95484e80-37df-11ea-80af-ff700b5b7d54.png)

**Comment by JKetelaar** [comment]
Fixed this bug, for some reason the Magento module was disabled.

Try executing: 
```
bin/magento module:enable Magento_Persistent
```

**Comment by JKetelaar** [comment]
Yet this does not resolve not being able to update to `1.0.3`.
It should work if you remove `tig/core` from the composer.json requirements

**Comment by macguydvd** [comment]
@JKetelaar thanks for the suggestion, when I run:

`bin/magento module:enable Magento_Persistent`

I receive the message:

`No modules were changed.`

So it wasn't disabled on my system.

**Comment by JKetelaar** [comment]
What if you disable all modules:
```
bin/magento module:status | grep -v Magento_ | grep -v "List of" | grep -v None | grep -v -e '^$'| xargs php bin/magento module:disable -f
```
And then check Stores -> Configuration -> Customers

**Comment by tig-svenluiks** [comment]
Hey @macguydvd,

Could you contact us at servicedesk@tig.nl? Then we will help you out further with the problem.

King regards,
Sven

**Comment by tig-dennisvanderhammen** [comment]
Hi @macguydvd,

We have released a version v1.0.4. Could you please try installing this version?
If you still encounter issues with the extension, please contact us at the e-mailadres my colleague mentioned, because then there might be something else going on, and we would like to help you finding the root cause of your issue!

For now I'll be closing this issue, as this isn't reproducible in v1.0.4.
